src/java.base/windows/native/libjli/java_md.h

```diff
@@ -42,9 +42,10 @@
 #define JLONG_FORMAT_SPECIFIER "%I64d"
 
 /*
- * Function prototypes.
+ * Windows specific declarations.
  */
 
+void reportWithLastWindowsError(const char* message, ...);
```

Comment:
Why does this need to be exported in the header file? Are you expecting other code to call this?

Reply:
I left that in there since it would possibly be a useful utility to have for other JLI code that might need to work with Windows errors in the future

Reply:
In that case shouldn't the `JLI_xxx` naming scheme be retained?